index.js

```diff
@@ -13,21 +15,37 @@ function ZipCodeApi() {
 
 ZipCodeApi.prototype.init = function(options) {
   options = options || {};
+  const clientKey = options.clientKey;
   const apiKey = options.apiKey;
+  const domain = options.domain;
 
-  if (!apiKey) {
-    throw new Error('Must set an API key');
+  if (!clientKey || !apiKey || !domain) {
```

Comment:
Is `domain` required?  I think we may want to support both options (and only hash if domain is present)

Reply:
@golmansax @caitlin I updated it to be backwards compatible and added some more tests...
https://github.com/boostmyschool/zipcodeapi/pull/1/commits/3f0493a19bcfd0440026e3fad965158956eb2b7b